src/main/java/FileStorage.java

```diff
@@ -0,0 +1,71 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.util.UUID;
+
+public class FileStorage implements Storage{
```

Comment:
you need to have a service, that is used to *store* an object. it's methods should not be dependent on implementation, and it should call database/file service (also known as repositories)